docs/advanced/addons/vmimport.md

```diff
@@ -109,79 +126,100 @@ stringData:
   networkMapping:
-  - sourceNetwork: "dvSwitch 1"
-    destinationNetwork: "default/vlan1"
-  - sourceNetwork: "dvSwitch 2"
-    destinationNetwork: "default/vlan2"
-  sourceCluster: 
+    - sourceNetwork: "dvSwitch 1"
+      destinationNetwork: "default/vlan1"
+    - sourceNetwork: "dvSwitch 2"
+      destinationNetwork: "default/vlan2"
+  sourceCluster:
     name: vcsim
     namespace: default
     kind: VmwareSource
     apiVersion: migration.harvesterhci.io/v1beta1
+  storageClass: "harvester-longhorn"
 ```
 
-This will trigger the controller to export the VM named "alpine-export-test" on the VMWare source cluster to be exported, processed and recreated into the harvester cluster
-
-This can take a while based on the size of the virtual machine, but users should see `VirtualMachineImages` created for each disk in the defined virtual machine.
-
-The list of items in `networkMapping` will define how the source network interfaces are mapped to the Harvester Networks.
-
-If a match is not found, each unmatched network interface is attached to the default `managementNetwork`.
-
-Once the virtual machine has been imported successfully, the object will reflect the status:
-
-```shell
-$ kubectl get virtualmachineimport.migration
-NAME                    STATUS
-alpine-export-test      virtualMachineRunning
-openstack-cirros-test   virtualMachineRunning
-
-```
-
-Similarly, users can define a VirtualMachineImport for an OpenStack source as well:
+This will trigger the controller to export the VM named "alpine-export-test"
+from the folder "/vm-folder" on the VMWare source cluster to be exported,
+processed and recreated into the harvester cluster
+</TabItem>
+<TabItem value="openstack" label="OpenStack">
 
 ```yaml
 apiVersion: migration.harvesterhci.io/v1beta1
 kind: VirtualMachineImport
 metadata:
   name: openstack-demo
   namespace: default
-spec: 
+spec:
   virtualMachineName: "openstack-demo" #Name or UUID for instance
+  folder: "/vm-folder"
   networkMapping:
-  - sourceNetwork: "shared"
-    destinationNetwork: "default/vlan1"
-  - sourceNetwork: "public"
-    destinationNetwork: "default/vlan2"
-  sourceCluster: 
+    - sourceNetwork: "shared"
+      destinationNetwork: "default/vlan1"
+    - sourceNetwork: "public"
+      destinationNetwork: "default/vlan2"
+  sourceCluster:
     name: devstack
     namespace: default
     kind: OpenstackSource
     apiVersion: migration.harvesterhci.io/v1beta1
+  storageClass: "harvester-longhorn"
 ```
 
-:::note 
-OpenStack allows users to have multiple instances with the same name. In such a scenario, users are advised to use the Instance ID. The reconciliation logic tries to perform a name-to-ID lookup when a name is used.
-:::
\ No newline at end of file
+:::note
+OpenStack allows users to have multiple instances with the same name. In such a
+scenario, users are advised to use the Instance ID. The reconciliation logic
+tries to perform a name-to-ID lookup when a name is used.
+:::
```

Comment:
This entire sentence is weird. The OpenStack docs you linked don't even mention an 'instance ID', but to me it sounds like the original sentence refers to the instance name, not the host name. The OpenStack docs are pretty ambiguous as well because they throw a 'server name' into the mix.
I think the comment from the code example gives a good hint to clear things up:
None of this refers to the host name, as that's a property that can change dynamically and will not necessarily be reflected in the OpenStack control plane.
'Name' refers to the 'instance name' from the OpenStack docs.
The 'instance ID' refers to the UUID, which isn't mentioned in the linked OpenStack docs. But I know that OpenStack uses a UUID to uniquely identify VMs on the command line.
@irishgordo Didn't you do some testing with OpenStack the other day? Can you help clear this up?